src/sql/src/parser.rs

```diff
@@ -102,6 +102,21 @@ impl<'a> ParserContext<'a> {
 
                     Keyword::DROP => self.parse_drop(),
 
+                    // TODO(LFC): Use "Keyword::USE" when we can upgrade to newer version of crate sqlparser.
+                    Keyword::NoKeyword if w.value.to_lowercase() == "use" => {
```

Comment:
Adds a constant for `use` string?

Reply:
It'll soon be deleted, when we can upgrade to newer version of sqlparser (after datafusion is upgraded)